spikes/lk_filesystem_kit/main.cpp

```diff
@@ -0,0 +1,115 @@
+// Leka - LekaOS
+// Copyright 2020 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+
+#include "drivers/BufferedSerial.h"
+#include "rtos/ThisThread.h"
+
+#include "FATFileSystem.h"
+#include "FileSystemKit.h"
+#include "HelloWorld.h"
+#include "LogKit.h"
+#include "SDBlockDevice.h"
+
+using namespace leka;
+using namespace std::chrono_literals;
+
+auto file				= FileSystemKit::File {};
+constexpr auto kTmpfile = std::to_array("/tmp/XXXXXX");
+auto filename			= std::array<char, L_tmpnam> {};
```

Comment:
ici tu peux remplacer par:

```suggestion
auto filename			= std::to_array("/fs/tmp/xxxxx");
```

et il faut donner un nom au fichier genre `test_spike_lk_filesystem_kit`

donc

```suggestion
auto filename			= std::to_array("/fs/tmp/test_spike_lk_filesystem_kit");
```